hedera-node/hedera-app/src/main/java/com/hedera/node/app/state/merkle/MerkleHederaState.java

```diff
@@ -112,10 +112,10 @@ public class MerkleHederaState extends PartialNaryMerkleInternal implements Merk
      */
     private static final long DO_NOT_USE_IN_REAL_LIFE_CLASS_ID = 0x0000deadbeef0000L;
 
-    private static final long CLASS_ID = 0x2de3ead3caf06392L;
+    //    private static final long CLASS_ID = 0x2de3ead3caf06392L;
     // Uncomment the following class ID to run a mono -> modular state migration
     // NOTE: also change class ID of ServicesState
-    //    private static final long CLASS_ID = 0x8e300b0dfdafbb1aL;
+    private static final long CLASS_ID = 0x8e300b0dfdafbb1aL;
```

Comment:
I don't think taht file need to be commited

Reply:
This won't matter.

Reply:
Why won't it matter? Seems like this will create duplicate class ids.

Reply:
Reverted to original class_id.